src/createApp.js

```diff
@@ -6,8 +6,8 @@ import install from "./install";
 
 export default function createApp(answers) {
   const projectName = answers.name;
-  const directory = path.resolve(process.cwd() + "\\" + projectName);
-  const source = path.resolve(__dirname, ("..\\packages\\" + answers.type));
+  const directory = path.resolve(process.cwd() + "/" + projectName);
+  const source = path.resolve(__dirname, ("../packages/" + answers.type));
```

Comment:
`path.resolve` can actually take care of slashes for us if we let it:

```js
const directory = path.resolve(process.cwd(), projectName);
const source = path.resolve(__dirname, "..", "packages", answers.type));
```

Reply:
cc @DalderupMaurice

Reply:
Didn't knew! 
I'll include this in the next PR - the CLI teel needs huge refactoring. I am currently working on it :D